src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -203,6 +235,9 @@ IConfigurationSource configurationSource
     public class StateVectorToHtmlResultEncoder : IResultEncoder
     {
         private const double TWO_PI = 2.0 * System.Math.PI;
+        private double count = 0.0;
+
+        private int measurement_off = 0; //when equal to 1 then if statement works
```

Comment:
For consistency with other C# variable names, I'd suggest `camelCase` here:
```suggestion
        private int measurementOffset = 0; //when equal to 1 then if statement works
```

Reply:
Deleted the variable because I didn't use it anywhere else